### DISCUSSION TITLE
ImageRun couldnt work

### DISCUSSION THREAD

**Discussion by ddhearly** [dolanmiu/docx#1350] [Ideas / Requests]
Im using <script src="https://unpkg.com/docx@5.0.2/build/index.js"></script> working to create docx using browser but ImageRun could not worked. Please help me to display image in docx. Thankyou

**Comment by dolanmiu** [comment]
Have you checked this example?

https://stackblitz.com/edit/react-docx-images

**Reply by Kiber-pank** [reply]
> Have you checked this example?
> 
> https://stackblitz.com/edit/react-docx-images

There is nothing in the example

**Reply by SujithaKakkan** [reply]
Any solution to this problem? Me too having same issue. Not able to add images to document

**Reply by Kiber-pank** [reply]
> Любое решение этой проблемы? У меня тоже такая же проблема. Не удается добавить и
I found a solution.
server:
```
  .get('/image/title', function (req, res) {
    res.sendFile(path.join(__dirname, 'public/assets/img/title.png'))
  })
```
client:
```
    const blob = await fetch(
        "/image/title"
    ).then(r => r.blob());
```

```
    const image = new docx.ImageRun({
        data: blob,
        transformation: {
            width: 780,
            height: 200,
        },
    });
```
